### PR TITLE
Login page rework

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it updates login page design.

**PR intended to be tested with API branch:** 3.0

### Screenshots
Zrzut ekranu 2021-12-20 o 10 16 33
Zrzut ekranu 2021-12-20 o 10 16 44

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Review by orzechdev — CHANGES_REQUESTED** [review]
Saleor logo should be in left top corner:
Zrzut ekranu 2021-12-20 o 11 03 12

File: src/auth/components/Layout.tsx [hunk 1]
```diff
@@ -76,18 +83,21 @@ const Layout: React.FC = props => {
...
-      </div>
       <div className={classes.mainPanel}>
         <div className={classes.mainPanelContent}>
           <SVG
             className={classes.logo}
             src={themeType === "dark" ? saleorDarkLogo : saleorLightLogo}
           />
           {children}
+          <footer className={classes.footer}>
+            ©2021 Saleor Commerce. All rights reserved
```
[the remark this answers is not in the document]
Comment (orzechdev) [reply src/auth/components/Layout.tsx:94]: @pwgryglak

[the remark this answers is not in the document]
Comment (orzechdev) [reply src/auth/components/Layout.tsx:94]: What do you think?

[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply src/auth/components/Layout.tsx:94]: > Saleor logo should be in left top corner: Zrzut ekranu 2021-12-20 o 11 03 12

How did you break it?

[the remark this answers is not in the document]
Comment (orzechdev) [reply src/auth/components/Layout.tsx:94]: @dominik-zeglen  It is the screenshot from Figma design

[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply src/auth/components/Layout.tsx:94]: oh

**Review by orzechdev — APPROVED** [review]

**Review by Droniu — APPROVED** [review]

**Pull request merged**